components/movementsensor/gpsrtkserial/gpsrtkserial_test.go

```diff
@@ -0,0 +1,177 @@
+package gpsrtkserial
+
+import (
+	"context"
+	"math"
+	"testing"
+
+	"github.com/edaniels/golog"
+	geo "github.com/kellydunn/golang-geo"
+	"go.viam.com/test"
+	"go.viam.com/utils"
+
+	"go.viam.com/rdk/components/movementsensor/fake"
+	rtk "go.viam.com/rdk/components/movementsensor/rtkutils"
+)
+
```

Comment:
all the same comments/questions I had in `gpsrtkpmtk_test.go` apply to this file as well